src/pages/Films/Films.tsx

```diff
@@ -33,20 +38,15 @@ function Films({ selectedGenre }: GenreProps) {
 	}, [selectedGenre]);
 
 	return (
-		<FilmsStyled>
-			<Carousel />
-		</FilmsStyled>
+		<S.Films>
+			<FilmsHeroImage src={heroImage} alt="hero image" />
```

Comment:
Trochę słaby alt, dla czytnków ekranowych i osób z niepełnosprawnością nic to totalnie nie mówi. To musi być jakiś description lub title filmu przynajniej z dopiskiem - okładka, albo coś.